### PR TITLE
FF88 RelNote: Presentation API removed

### PR DESCRIPTION
**Pull request by hamishwillee**
The presentation API has been removed as per https://bugzilla.mozilla.org/show_bug.cgi?id=1697680. It is not deprecated, there was just concern that the code did not actually work properly, and that there was no priority to making sure it did work properly. 

I haven't attempted to explain the reasons - just added the fact and linked to the defect. The BCD provides more detailed support information, including the fact that it hasn't worked on FF Android for a few releases either.

Part of fixing #3461

### CODE REVIEW

File: files/en-us/web/api/presentation_api/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
...
 
-<p><span class="seoSummary">The Presentation API lets a {{Glossary("user agent")}} (such as a Web browser) effectively display web content through large presentation devices such as projectors and network-connected televisions.</span> Supported types of multimedia devices include both displays which are wired using HDMI, DVI, or the like, or wireless, using <a href="https://www.dlna.org/">DLNA</a>, <a href="/en-US/docs/">Chromecast</a>, <a href="https://developer.apple.com/airplay/">AirPlay</a>, or <a href="https://www.wi-fi.org/discover-wi-fi/miracast">Miracast</a>.</p>
+<p><span class="seoSummary">The Presentation API lets a {{Glossary("user agent")}} (such as a Web browser) effectively display web content through large presentation devices such as projectors and network-connected televisions.</span> Supported types of multimedia devices include both displays which are wired using HDMI, DVI, or the like, or wireless, using <a href="https://www.dlna.org/">DLNA</a>, <a href="/en-US/docs/Web">Chromecast</a>, <a href="https://developer.apple.com/airplay/">AirPlay</a>, or <a href="https://www.wi-fi.org/discover-wi-fi/miracast">Miracast</a>.</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/api/presentation_api/index.html:13]: Chromecast URL here is definitely wrong ;-)

**Review by chrisdavidmills — APPROVED** [review]
Thanks @hamishwillee . The rel note looks fine; just one review comment to address. Thanks!

File: files/en-us/web/api/presentation_api/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/api/presentation_api/index.html]: Good catch - not the best automatic "Flaw fix" ever. I have modified to point to the wikipedia page. Merge at will :-)

Comment (hamishwillee) [reply files/en-us/web/api/presentation_api/index.html]: Actually, fixed back to what it was.

**Review by chrisdavidmills — APPROVED** [review]
Perfect, let's get this merged.

**Pull request merged**